doc/nrf/releases_and_maturity/releases/release-notes-3.1.0-preview1.rst

```diff
@@ -0,0 +1,652 @@
+
+IDE, OS, and tool support
+=========================
+
+|no_changes_yet_note|
+
+Board support
+=============
+
+|no_changes_yet_note|
+
+Build and configuration system
+==============================
+
+|no_changes_yet_note|
+
+Bootloaders and DFU
+===================
+
+|no_changes_yet_note|
+
+Developing with nRF91 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with nRF70 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with nRF54L Series
+=============================
+
+|no_changes_yet_note|
+
+Developing with nRF54H Series
+=============================
+
+|no_changes_yet_note|
+
+Developing with nRF53 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with nRF52 Series
+============================
+
+|no_changes_yet_note|
+
+Developing with Thingy:91 X
+===========================
+
+|no_changes_yet_note|
+
+Developing with Thingy:91
+=========================
+
+|no_changes_yet_note|
+
+Developing with Thingy:53
+=========================
+
+|no_changes_yet_note|
+
+Developing with PMICs
+=====================
+
+|no_changes_yet_note|
+
+Developing with Front-End Modules
+=================================
+
+* Added the temperature compensation feature for the nRF2220 Front-End Module.
+
+Developing with custom boards
+=============================
+
+|no_changes_yet_note|
+
+Security
+========
+
+|no_changes_yet_note|
+
+Protocols
+=========
+
+|no_changes_yet_note|
+
+Amazon Sidewalk
+---------------
+
+|no_changes_yet_note|
+
+Bluetooth® LE
+-------------
+
+|no_changes_yet_note|
+
+Bluetooth Mesh
+--------------
+
+|no_changes_yet_note|
+
+DECT NR+
+--------
+
+|no_changes_yet_note|
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+|no_changes_yet_note|
+
+Gazell
+------
+
+|no_changes_yet_note|
+
+Matter
+------
+
+* Updated:
+
+  * The ``west zap-generate`` command to remove previously generated ZAP files before generating new files.
+    To skip removing the files, use the ``--keep-previous`` argument.
+
+Matter fork
++++++++++++
+
+|no_changes_yet_note|
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+|no_changes_yet_note|
+
+Thread
+------
+
+|no_changes_yet_note|
+
+
+Wi-Fi®
+------
+
+|no_changes_yet_note|
+
+Applications
+============
+
+|no_changes_yet_note|
+
+Connectivity bridge
+-------------------
+
+|no_changes_yet_note|
+
+IPC radio firmware
+------------------
+
+|no_changes_yet_note|
+
+Matter bridge
+-------------
+
+|no_changes_yet_note|
+
+nRF5340 Audio
+-------------
+* Updated:
+
+  * The application to use the ``NFC.TAGHEADER0`` value from FICR as the broadcast ID instead of using a random ID.
+
+nRF Desktop
+-----------
+
+* Updated the application configurations for dongles on memory-limited SoCs (nRF52820) to reuse system workqueue for GATT Discovery Manager (:kconfig:option:`CONFIG_BT_GATT_DM_WORKQ_SYS`).
+  This change helps to reduce RAM usage.
```

Comment:
```suggestion
* Updated the application configurations for dongles on memory-limited SoCs (nRF52820) to reuse the system workqueue for GATT Discovery Manager (:kconfig:option:`CONFIG_BT_GATT_DM_WORKQ_SYS`).
  This helps to reduce RAM usage.
```
I think I had this suggestion in the original PR including this changelog entry.

Reply:
Updated the changelog as well.